### PR TITLE
Make String.toRFC3986 and String.capFLetter normal functions

### PR DESCRIPTION
**Pull request by wil93**
Also a minor tweak to _isMobile() method

See issue #265

### CODE REVIEW

**Comment by hhsnopek** [comment]
Hey @wil93 would be so kind to remove the functions and create a StringUtils with those as methods? This way we'd leave a smaller footprint and not invade the global namespace with the functions :grinning:

**Comment by wil93** [comment]
Done :smile:

**Comment by hhsnopek** [comment]
You rock @wil93 :smile: - tho could you change the commit message? haha they're no longer functions - I'll merge in the morning :sunrise:

File: src/share-utils.js [hunk 1]
```diff
@@ -275,36 +277,9 @@ class ShareUtils {
    * @private
    */
   _isMobile() {
-    if(navigator.userAgent.match(/Android|iPhone|PhantomJS/i) &&
-       !navigator.userAgent.match(/iPod|iPad/i))
-      return true;
-    return false;
+    return navigator.userAgent.match(/Android|iPhone|PhantomJS/i) &&
+           !navigator.userAgent.match(/iPod|iPad/i);
```
Comment (hhsnopek) [inline src/share-utils.js]: could you make this a separate commit as well? - sorry for being picky 🙃

Comment (hhsnopek) [inline src/share-utils.js]: could you make this a separate commit as well? - sorry for being picky 🙃

**Comment by wil93** [comment]
Whoops, of course, done :smile:

File: src/share-utils.js (hunk 1 above)
Comment (wil93) [inline src/share-utils.js]: No problem :+1:

**Comment by hhsnopek** [comment]
Awesome - thank you again @wil93 
I'll be dropping a release tomorrow morning :smile:
